### PR TITLE
Feature#31/style the post view

### PR DESCRIPTION
**Pull request by Javi1499**
# Description
What did you implement/Why did you implement this?
   - Create parent elements for post view to display information like product name and description
   - Added styles based on figma design.
   - Added responsive design to display the view on different devices without losing quality and layout
  ## Screenshots 
imagen
imagen

### CODE REVIEW

**Review by EliasUR — DISMISSED** [review]
LGTM

**Review by germanruzca — DISMISSED** [review]
LGTM

File: app/assets/stylesheets/base/_globals.scss [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
   padding: 0;
 }
+
+.main-container {
+  padding-bottom: 260px;
+
+  @media screen and (max-width: $mobile) {
+    padding-bottom: 710px;
+  }
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/base/_globals.scss:20]: Add a blank line

File: app/assets/stylesheets/components/_image.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.image{
+  width: 100%;
+  height: 100%;
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/components/_image.scss:4]: Add a blank line

File: app/assets/stylesheets/pages/_post.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  &__features {
+    grid-area: features;
+  }
+
+  &__reviews {
+    grid-area: reviews;
+    height: 300px; //this height is only for test
+    text-align: center;
+  }
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/pages/_post.scss:44]: Add a blank line

File: app/views/home/_user_navbar.html.erb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <div class="user_navbar">
   <div class="user_navbar__box">
-    <%= image_tag('test.png', class: "user_navbar__image") %>
+    <%#= image_tag('test.png', class: "user_navbar__image") %>
```
Comment (YilKanda) [inline app/views/home/_user_navbar.html.erb:3]: Code that you do not use, delete it.

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -12,11 +12,16 @@
...
-    <main>
-      <p class="notice"><%= notice %></p>
-      <p class="alert"><%= alert %></p>
-      <%= yield %>
-      <%=render 'layouts/footer' %>
-    </main>  
+  <main>
+    <!--      <p class="notice"><%#= notice %></p>-->
+    <!--      <p class="alert"><%#= alert %></p>-->
+
```
Comment (YilKanda) [inline app/views/layouts/application.html.erb:18]: Code that you do not use, delete it.

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -12,11 +12,16 @@
...
+  </main>
+  <div class="main-container">
+    <%= yield %>
+  </div>
+
```
Comment (YilKanda) [inline app/views/layouts/application.html.erb:23]: Eliminate extra space

File: app/views/posts/_post_info.html.erb [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+</div>
+<div class="info__detail">
+  <i class="fa-solid fa-truck info__icon--green"></i>
+  <p class="info__text info__text--green">Punto medio disponible</p>
+</div>
+<div class="info__actions">
+  <%= link_to "Comprar", edit_post_path(@post), class:"info__button info__button--dark" %>
+  <%= link_to "Agregar al carrito", edit_post_path(@post), class:"info__button info__button--light" %>
+</div>
+</div>
```
Comment (YilKanda) [inline app/views/posts/_post_info.html.erb:22]: Check the indentation of this file, add a blank line.

File: app/assets/stylesheets/pages/_post.scss [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    width: 100%;
+  }
+
+  &__features {
+    grid-area: features;
+  }
+
+  &__reviews {
+    grid-area: reviews;
+    height: 300px; //this height is only for test
```
Comment (YilKanda) [inline app/assets/stylesheets/pages/_post.scss:41]: If this item isn't useful for production, it's best to remove it when pushing it to GitHub.

Remember, you should not leave test or debug values ​​on GitHub.

Your code should always be as clean as possible.

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by germanruzca — APPROVED** [review]
LGTM

**Pull request merged**